src/Query/index.ts

```diff
@@ -256,7 +257,7 @@ export class Query {
     return agg;
```

Comment:
minor, can we specify return type for this fn to be mdb.Document[] as well. used for `aggregate(agg)` function